src/layers/legacy/medCoreLegacy/readers/medAbstractDataReader.cpp

```diff
@@ -0,0 +1,28 @@
+#include "medAbstractDataReader.h"
```

Comment:
the header is necessary. The last version can be found [here](https://github.com/medInria/medInria-public/blob/master/src/app/medInria/medExportVideoDialog.h).

Reply:
Done